docs/jwst/user_documentation/datamodels.rst

```diff
@@ -39,3 +39,8 @@ By default, when running in Python, the corrected data will be returned in-memor
 as a ``DataModel`` instead of being written as an output file.
 See :ref:`controlling output file behavior`<python_outputs>` for instructions on
 how to write the returned ``DataModel`` to an output file.
+
+API
+===
+
+.. automodapi:: jwst.datamodels
```

Comment:
I think I need this for the `DataContainer` API ref link to work.

Reply:
OK this opens a different can of worms... I will revert this and just disable the ref link.